### ISSUE TITLE
copay wallet empty

### ISSUE THREAD

**Issue by Ladoie** [bitpay/copay#11331]
Hi,
dont know if this is right place to post but ill still try. 

couple years ago 2-3 years ago I bought BTC and stored it to copay app on my iPhone. since than I did not used copay app. today I restored my wallet on my iPhone with 12 words, and the restored wallet showed balance 0. 
I cant find any history of my transactions and cant figure out where my BTC went. 

copay version is 11.0.4

I hope to find an answer and solution to get my bitcoins back. 

thank you 
Ladoie

**Comment by matiu** [comment]
Hi. can you provide an address of the wallet so we can check it out?

**Comment by Ladoie** [comment]
14WTDgjnDVfcysdDFNMZeC9roGnqV6GuaG

**Comment by matiu** [comment]
Hi, there is no transaction registered on the blockchain for that address o any other address of that wallet, are you sure you are using the right backup?

**Comment by Ladoie** [comment]
I just checked address where I have stored BTC. here it is 1PWnNWSJwr48WSWatzQ24icJ8CrpWJDoKE. can you tell me if this is copay wallet?

thank you

**Comment by travisb27** [comment]
Same problem

**Comment by Ladoie** [comment]
I have never created two wallets. attached is screen from coinbase transfer. 
<img width="448" alt="Screen Shot 2020-12-17 at 12 13 54 AM" src="https://user-images.githubusercontent.com/75463653/102447034-2bdb4c00-3ffd-11eb-984d-d7dfd61c16d8.png">

**Comment by ariazevedopt** [comment]
Same happened to me. Please help!

**Comment by rrlower** [comment]
im on the same spot, i wrote to bitpay to see if they help me,

**Comment by anywalletrecovery** [comment]
> im on the same spot, i wrote to bitpay to see if they help me,


yes this is very common issue with older versions of some wallets. I have recovered a few wallet like this before. however, not all of them are recoverable and the funds are lost for good

**Comment by isieditors** [comment]
i

> 14WTDgjnDVfcysdDFNMZeC9roGnqV6GuaG

and the address you have mentioned hasn't received any btc yet, you have made mistake somewhere

**Comment by johnjames333** [comment]
@rrlower 
@ariazevedopt 
@travisb27 
@Ladoie 
we have developed our own hardware for cracking all versions of bitcoin wallet password with superfast speed
and if you still has problem I can help you run it. you can find my contact details on my profile
or you can send me message in telegram: @johnjames333 or gmail : walletsrecoveryservice@gmail.com
